src/ucs/sys/sys.h

```diff
@@ -24,7 +24,9 @@
 #include <sys/types.h>
 #include <sys/uio.h>
 #include <sys/fcntl.h>
+#ifdef HAVE_SYS_EPOLL_H
 #include <sys/epoll.h>
+#endif
```

Comment:
I moved this `#include` to `src/ucp/core/ucp_worker.c` for minimizing dependency.
Please see #3610